src/platform/site-wide/side-nav/tests/01-accessibility.e2e.spec.js

```diff
@@ -0,0 +1,134 @@
+const E2eHelpers = require('../../../testing/e2e/helpers');
+
+module.exports = E2eHelpers.createE2eTest(client => {
+  client.openUrl(`${E2eHelpers.baseUrl}/pittsburgh-health-care/`);
+  E2eHelpers.overrideSmoothScrolling(client);
+  client.resizeWindow(1600, 1200);
```

Comment:
I don't think we have any but it would be cool to have shared constants for the screen sizes

Reply:
Oh that was just for local testing, no need for screen change on this test.